pipelines/internal/common/common.go

```diff
@@ -51,3 +55,29 @@ func (m *MapFlagValue) Set(input string) error {
 	}
 	return nil
 }
+
+// PipelineExecutionError is an error returned by the Genomics API
+// during a pipeline execution
+type PipelineExecutionError genomics.Status
+
+func (status PipelineExecutionError) Error() string {
+	return fmt.Sprintf( "executing pipeline: %d : %s", status.Code, status.Message)
+}
+
+// Map that indicates which Genomics errors are fatal and
```

Comment:
No need for the comment, the map is self explanatory

There also shouldn't be 'false' entries since you don't include everything... In fact, if there are only two we want to retry, just test them inline below:

func (err PipelineExecutionError) IsRetriable() bool {
  switch code.Code_name[int32(err.Code)] {
    case "UNAVAILABLE", "ABORTED":
      return true
    default:
      return false
  }
}